stylesheet.css

```diff
@@ -97,3 +97,371 @@
+
+.project {
+  display: flex;
+  background-image: url(./images/Profesional\ Printing.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  flex-direction: column;
+}
+
+.project_title {
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  color: #fff;
+  margin-top: 80px;
+  margin-bottom: 0;
+  margin-left: 15px;
+}
+
+.project_description {
+  width: 301px;
+  height: 99px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: 15px;
+  margin-bottom: 0;
+}
+
+.project_tools {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  order: 2;
+  height: 40px;
+  margin-right: 15px;
+  padding: 0;
+}
+
+.tool {
+  display: flex;
+  color: #fff;
+  width: 71px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin-left: 15px;
+  align-items: center;
+  justify-content: center;
+}
+
+.project_link {
+  display: flex;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  order: 3;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  justify-content: center;
+  letter-spacing: 0.03em;
+}
+
+.about_me {
+  background-color: #1c1a19;
+  display: flex;
+  min-width: 100vw;
+  min-height: fit-content;
+  flex-direction: column;
+}
+
+.about_me_intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 80vh;
+  background-image: url(./images/Background_aboutme.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  order: 0;
+}
+
+.about_me_title {
+  width: 327px;
+  height: 79px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.description1 {
+  width: 327px;
+  height: 148px;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.resume {
+  background-color: #ff6b00;
+  order: 2;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  align-self: flex-start;
+  width: 156px;
+  height: 48px;
+  margin-top: 15px;
+  margin-left: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
```

Comment:
- [ ] I would kindly suggest you work on the button `Get resumi` and let it aligned with other text as seen in the Figma design. 
- [ ] Also I would kindly advice you give spaces between the the cards like margin top and bottom
- [ ] Also kindly give all the buttons some kind of effect like `:hover, :active and :enabled`

Your Design :point_down: 

![Screenshot_20230610_230603](https://github.com/ommarz/Personal_Portfolio/assets/78634841/bcb0905d-89f6-493c-a470-9c394446f136)